sql/src/test/java/io/crate/execution/ddl/RerouteActionsTest.java

```diff
@@ -1,219 +0,0 @@
-/*
- * Licensed to Crate under one or more contributor license agreements.
- * See the NOTICE file distributed with this work for additional
- * information regarding copyright ownership.  Crate licenses this file
- * to you under the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.  You may
- * obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
- * implied.  See the License for the specific language governing
- * permissions and limitations under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial
- * agreement.
- */
-
-package io.crate.execution.ddl;
-
-import io.crate.analyze.RerouteAllocateReplicaShardAnalyzedStatement;
-import io.crate.analyze.RerouteCancelShardAnalyzedStatement;
-import io.crate.analyze.RerouteMoveShardAnalyzedStatement;
-import io.crate.data.Row;
-import io.crate.metadata.RelationName;
-import io.crate.metadata.blob.BlobTableInfo;
-import io.crate.metadata.doc.DocTableInfo;
-import io.crate.sql.parser.SqlParser;
-import io.crate.sql.tree.Assignment;
-import io.crate.sql.tree.BooleanLiteral;
-import io.crate.sql.tree.GenericProperties;
-import io.crate.sql.tree.GenericProperty;
-import io.crate.sql.tree.QualifiedName;
-import io.crate.sql.tree.QualifiedNameReference;
-import io.crate.sql.tree.StringLiteral;
-import io.crate.test.integration.CrateDummyClusterServiceUnitTest;
-import io.crate.testing.SQLExecutor;
-import org.elasticsearch.action.admin.cluster.reroute.ClusterRerouteRequest;
-import org.elasticsearch.cluster.routing.allocation.command.AllocateReplicaAllocationCommand;
-import org.elasticsearch.cluster.routing.allocation.command.CancelAllocationCommand;
-import org.elasticsearch.cluster.routing.allocation.command.MoveAllocationCommand;
-import org.elasticsearch.common.Randomness;
-import org.junit.Before;
-import org.junit.Test;
-
-import java.util.Arrays;
-import java.util.Collections;
-
-import static io.crate.analyze.TableDefinitions.TEST_PARTITIONED_TABLE_DEFINITION;
-import static io.crate.analyze.TableDefinitions.TEST_PARTITIONED_TABLE_PARTITIONS;
-import static io.crate.analyze.TableDefinitions.USER_TABLE_DEFINITION;
-import static org.hamcrest.Matchers.is;
-
-public class RerouteActionsTest extends CrateDummyClusterServiceUnitTest {
```

Comment:
`RerouteActions` and `RerouteActionsTest` are removed.

It contained methods like `prepareMoveShardReq`, `prepareAllocateReplicaReq`, etc. that used to analyze expressions and evaluate them. These methods are not actually needed any longer with the new `expr -> symnbol -> obj` pattern we follow.

It is also safe to delete these tests. The tests cases are covered with the analyzer tests.